tests/SBRPTests/SBRPFixture.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace SBRPTests;
+
+public class SBRPFixture
+{
+    public string WorkingDirectory { get; set; }
+
+    public SBRPFixture()
+    {
+        WorkingDirectory = Environment.CurrentDirectory = Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.IndexOf("source-build-reference-packages") + 31);
+        CommandHelper.CommandOutput("git", "reset HEAD", WorkingDirectory);
+        CommandHelper.CommandOutput("git", "checkout src/", WorkingDirectory);
```

Comment:
Won't this lose uncommitted changes on disk?  My initial thought was the tests should utilize a sandboxed location for the generated output and compare it against the actual content.  It is hypothetically possible that we could overwrite uncommitted user customization to the package we are testing against.

Reply:
The original request I got is "Create some light-weight, basic, tests, using XUnit. For the start, we’d want to have few basic tests that just generate existing reference packages and use git to see if there are any open files". So I need to do this cleanup between different tests. I'll discuss with @NikolaMilosavljevic to see if we need to change current implementation

Reply:
My primary concern is that the test can't revert changes the user may have locally.

Reply:
This would be easily achievable using `-d` option of the generation script.

However, if user has uncommitted customization changes in the exact same project, the test will fail. Tests might also be invalid for any project that was modified after original generation.

An alternative approach might be to check for any uncommitted changes and either 1) not run tests until those are committed, or 2) stash and later reapply the changes.

Reply:
@v-chayan running tests in sandbox is preferable. We should not revert any user's changes. We could have a warning that there are uncommitted changes in the projects that we're regenerating.

Reply:
Got it. Will update the implementation